riscv/insns/prefetch_w.h

```diff
@@ -0,0 +1,2 @@
+require_extension(EXT_ZICBOP);
```

Comment:
Delete this line since this instruction is a hint instruction and *must not* generate illegal instruction trap.